_security-plugin/access-control/document-level-security.md

```diff
@@ -124,3 +124,22 @@ PUT _plugins/_security/api/roles/abac
   }]
 }
 ```
+## Term Lookup Query (TLQ) modes
+
+You can perform term lookup queries (TQLs) with Document-level security using either of two DLS modes: Lucene-level or Filter-level. By default, OpenSearch Security will detect if a DLS query contains a TLQ or not, and chooses the appropriate mode automatically at runtime.
```

Comment:
Typo of TQL here? Also, I think we avoid referring to plugins like OpenSearch {plugin name}, so we might want to go with: "By default, the security plugin detects if a DLS query contains a TLQ..."

Reply:
Thanks!